src/Tools/ExternalAccess/FSharp/FSharpGlyphHelpersObsolete.cs

```diff
@@ -0,0 +1,321 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.FSharp
+{
+    internal static class FSharpGlyphHelpersObsolete
+    {
+        [Obsolete("Only used to allow IVTs to work temporarily, will be removed when IVTs are fully removed.")]
+        public static Microsoft.CodeAnalysis.Glyph Convert(FSharpGlyph glyph)
+        {
+            switch (glyph)
```

Comment:
This block would be 1/3 the size by using a switch expression:

```csharp
return glyph switch
{
    FSharpGlyph.None => Microsoft.CodeAnalysis.Glyph.None,
    FSharpGlyph.Assembly => Microsoft.CodeAnalysis.Glyph.Assembly,
    ...
};
```

No real need to change it until such time as we have a refactoring in place to make it automatic. 😄